crates/bevy_ecs/src/component.rs

```diff
@@ -12,26 +12,85 @@ use std::{
     any::{Any, TypeId},
 };
 
-/// A component is data associated with an [`Entity`](crate::entity::Entity). Each entity can have
-/// multiple different types of components, but only one of them per type.
+// TODO: The content in the section “Implementing the trait for foreign types” may be more appropriate
```

Comment:
Yes, agreed. Can you a) file an issue on the bevy-website repo to migrate this so we don't forget and then b) remove this TODO

Reply:
- Issue opened: bevyengine/bevy-website#375.
- TODO removed.